src/dotnet/APIView/APIView/TreeRendererAPIV.cs

```diff
@@ -0,0 +1,220 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace APIView
+{
+    public class TreeRendererAPIV
+    {
+        private const int IndentSize = 4;
+
+        public string Render(AssemblyAPIV assembly)
+        {
+            return Render(assembly.GlobalNamespace);
+        }
+
+        private string Render(EventAPIV e, int indents = 0)
+        {
+            string indent = new string(' ', indents * IndentSize);
+
+            //TODO: determine whether event is EventHandler or other type - and if it has type parameter(s)
+            return indent + "public event EventHandler " + e.Name + ";";
+        }
+
+        private string Render(FieldAPIV f, int indents = 0)
+        {
+            string indent = new string(' ', indents * IndentSize);
+
+            StringBuilder returnString = new StringBuilder(indent + "public");
```

Comment:
Don't create a StringBuilder in every method, create on in `Render(AssemblyAPIV assembly)` and pass it around as parameter